mon-pix/tests/integration/components/navbar-header-test.js

```diff
@@ -37,15 +37,16 @@ describe('Integration | Component | navbar-header', function() {
           }
         }
       });
-      await render(hbs`{{navbar-header media=media burger=burger }}`);
+      await render(hbs`<NavbarHeader @burger={{this.burger}}></NavbarHeader>`);
       // then
       expect(find('.navbar-mobile-header__container')).to.exist;
       expect(find('.navbar-mobile-header__burger-icon')).to.exist;
     });
 
     it('should be rendered in mobile/tablet mode without burger', async function() {
       // when
-      await render(hbs`{{navbar-header media=media}}`);
+      await render(hbs`<NavbarHeader></NavbarHeader>`);
```

Comment:
En one line, j'ai tendance à préférer qui me parait plus lisible
```suggestion
      await render(hbs`<NavbarHeader />`);
```

Reply:
Je ne sais pas s'il y a une convention pour cela, mais je préfère les tag HTML qui se ferment clairement, d'où `<NavbarHeader></NavbarHeader>`. Après sur Pix, j'ai l'impression que la tendance est plutôt à écrire `<NavbarHeader />`

Reply:
Il existe aussi des balises auto-fermantes en HTML comme `img` par exemple. ça me surprend pas je suis plus pour la version auto-fermante quand il n'y pas de contenu ça permet d'identifier ça plus rapidement à mon avis.